larq/quantized_variable.py

```diff
@@ -39,6 +39,63 @@ def __init__(self, variable, quantizer=None, precision=None):
         self.quantizer = quantizer
         self.precision = precision or getattr(quantizer, "precision", None)
 
+    @classmethod
```

Comment:
We could also use a `staticmethod` here, not sure what's better.

Reply:
The [`tf.data.Dataset.from_tensor`](https://github.com/tensorflow/tensorflow/blob/r2.0/tensorflow/python/data/ops/dataset_ops.py#L397) and [`tf.data.Dataset.from_tensor_slices`](https://github.com/tensorflow/tensorflow/blob/r2.0/tensorflow/python/data/ops/dataset_ops.py#L417) factories both use `staticmethod`, but reading [this StackOverflow answer](https://stackoverflow.com/a/15031999/1814949) and [this Software Engineering StackExchange answer](https://softwareengineering.stackexchange.com/a/166715), I think that you're right in using a `classmethod`. Maybe we should actually PR those TF Datasets methods to turn them into `classmethod`. 😄

Reply:
Cool, thanks for checking :+1: